governance/polls/Resilience Research Project Application - Legal Opinion on MiCA Compliance for CEX Listings of USDS and DAI - December 30, 2024.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Resilience Research Project Application - Legal Opinion on MiCA Compliance for CEX Listings of USDS and DAI - December 30, 2024
+summary: Signal your support or opposition to the listed Resilience Research Project Application.
+discussion_link: https://forum.sky.money/t/resilience-research-project-application-legal-opinion-on-mica-compliance-for-cex-listings-of-usds-and-dai/25759
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes - Proceed with obtaining formal legal expertise to ensure compliance with MiCA.
+   2: No - Do not proceed with formal legal consultation.
+start_date: 2024-12-30T16:00:00
+end_date: 2025-01-02T16:00:00
+---
+# Poll: Resilience Research Project Application - Legal Opinion on MiCA Compliance for CEX Listings of USDS and DAI - December 30, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Support Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.1_Operational_Weekly_Cycle-b189fa17-57a9-4d4e-9780-0ce4efd94211%7C0db30308) will be active for three days beginning on Monday, December 30 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to funding this Resilience Research Project Application which seeks to fund a legal opinion on MiCA compliance for CEX listings of USDS and Dai. The project abstract is as follows:
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to funding a Resilience Research Project proposal which seeks to produce a legal opinion on MiCA compliance for CEX listings of USDS and Dai. The project abstract is as follows:
```
This language is closer to [what stands in the Atlas](https://sky-atlas.powerhouse.io/A.2.8.1.4_Application_Process/9660eeb7-900e-49ff-9323-e7560b7a6436%7C9e1f609f13e1), I think.